roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -3,20 +3,25 @@
 from pathlib import PosixPath
 import sys
 import json
+import argparse
+import os
 
-job_prefix = sys.argv[1]
-if len(sys.argv) == 3:
-    targets_from_cli = sys.argv[2].split(" ")
-else:
-    targets_from_cli = []
-jobs = [f"{job_prefix}{i}" for i in range(10)]
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-t', '--targets', help="list of targets for integration testing. example: 'ec2_tag ami_info ec2_instance'",default="")
+parser.add_argument('-p', '--prefix', help='list of targets for integration testing', default="job_")
+parser.add_argument('-c', '--collection_path', help='path to the collection.', default=os.getcwd())
```

Comment:
```suggestion
parser.add_argument('-c', '--collection_path', help='path to the collection.', default=os.getcwd())
parser.add_argument('-j', '--jobs', help='total number of jobs to split tests over.', default=10)
```

While we're at it we should move total_jobs to an arg too.  community.aws has a *lot* more modules than amazon.aws it's only a matter of time before the necessary number of jobs differs between the two.